modules/authentication/src/handlers/facebook/facebook.ts

```diff
@@ -93,8 +93,11 @@ export class FacebookHandlers extends OAuth2<Payload, FacebookSettings> {
       },
       new ConduitRouteReturnDefinition('FacebookResponse', {
         userId: ConduitString.Required,
-        accessToken: ConduitString.Required,
-        refreshToken: ConduitString.Required,
+        accessToken: ConduitString.Optional,
+        refreshToken: ConduitString.Optional,
+        message: ConduitString.Optional,
```

Comment:
set/removeCookies is not a response as I've said yesterday. Remove it and also remove the message.